api/queries/community/memberGrowth.js

```diff
@@ -30,7 +30,7 @@ export default async (
   }
 
   return {
-    count: await getMemberCount(id),
+    count: memberCount || 1,
```

Comment:
Should fallback to `await getMemberCount(id)` just to make sure?

Reply:
Feels hacky, as though we don't trust the migration or the logic in the code to keep this right. Wdyt? Even having the `|| 1` feels brittle, but is a worst-case fallback right?

Reply:
It just feels like this is going to be confusing to us in the future: "why does this one specific resolver fall from a object field fall back to a full db query?"

Just seems like we're overdoing this resolver with code that feels overly specific and will be confusing to maintain in the future. Technically it's not bad, but we'd also want to know if it ever had to run the fallback code right? That would mean we have bad db data